docs/developer/app-store/apps/segment.mdx

```diff
@@ -67,8 +67,8 @@ Example of successful handling of `ORDER_CREATED` webhook:
 ```mermaid
 sequenceDiagram
     OrderCreatedWebhook->>+App: Send webhook payload
-    App-->>-AppMetadata: Fetch app configuration from metadata
-    AppMetadata-->>+App: Return configuration
+    App-->>-AppConfiguration: Fetch app configuration
+    AppConfiguration-->>+App: Return configuration
```

Comment:
I think this is a wrong abstract. Configuration is something internal to app, contrary to previous approach where metadata *was* something external.

I suggest to change AppConfiguration to `Database` and thats it - configuration is indeed fetched from DB
The details about DynamoDB can be left to "development" part